cypress/integration/userCanAssignKarmaPointsUpToBalance.feature.js

```diff
@@ -0,0 +1,91 @@
+describe("User can assign Karma Points to the extent of their balance", () => {
+  beforeEach(() => {
+    cy.server();
+  });
+
+  describe("can log in successfully", () => {
```

Comment:
This is not the scope of the feature, it can just be done in the beforeEach. In the next PR I added this to cypress commands.js, so that you can write `cy.login()` in the beforeEach block: 
```
Cypress.Commands.add("login", () => {
  cy.route({
    method: "POST",
    url: "**/auth/*",
    response: "fixture:successful_login.json",
    headers: {
      uid: "user@mail.com",
    },
  });
  cy.route({
    method: "GET",
    url: "**/auth/*",
    response: "fixture:successful_login.json",
    headers: {
      uid: "user@mail.com",
    },
  });

  cy.visit("/login");
  cy.get("#login-form").within(() => {
    cy.get("#email").type("user@mail.com");
    cy.get("#password").type("password");
    cy.get("Button").contains("Submit").click();
  });
})
```

Reply:
thx